### PR TITLE
Asuswrt config entries

### PR DESCRIPTION
**Pull request by ollo69**
## Proposed change

AsusWRT documentation update to add ConfgFlow and related options. 
The associated PR in the codebase use a dedicated branch and will require multiple commit before merging in dev branch. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/46468
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/asuswrt.markdown
Comment (frenck) [inline source/_integrations/asuswrt.markdown]: YAML configuration should be removed from documentation when the configuration flow is introduced.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by MartinHjelmare** [comment]
Parent PR is merged into a side branch (non default). More core PRs are expected to come with this docs PR linked.

**Comment by MartinHjelmare** [comment]
@ollo69 please let us know the status of the parent PRs when all are ready. Ie when the big PR from the separate side branch is merged into core dev branch, ping us here so we know when to merge this.

**Comment by ollo69** [comment]
Just to understand, the PR to merge the branch `asuswrt-config-entries` in `dev` must be created by me? Sorry but I'm not expert in this process.

**Comment by MartinHjelmare** [comment]
Yes, that's correct. 👍

**Comment by MartinHjelmare** [comment]
Parent PR was merged.

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**
